types/express-jwt/index.d.ts

```diff
@@ -62,3 +62,14 @@ declare namespace jwt {
         constructor(code: ErrorCode, error: { message: string });
     }
 }
+
+declare global {
+    namespace Express {
+        // tslint:disable-next-line:no-empty-interface
+        interface User {}
+
+        interface Request {
+            user: User;
+        }
+    }
+}
```

Comment:
This seems to match the behavior in https://github.com/auth0/express-jwt/blob/master/README.md#usage 👍🏻

Reply:
this also matches the established convention, e.g.:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/passport/index.d.ts#L19-L24